test/e2e/operator/garden/create_delete.go

```diff
@@ -160,7 +160,18 @@ var _ = Describe("Garden Tests", Label("Garden", "default"), func() {
 			g.Expect(controllerRegistrationList.Items).To(ContainElement(MatchFields(IgnoreExtras, Fields{"ObjectMeta": MatchFields(IgnoreExtras, Fields{"Name": Equal("provider-local")})})))
 			controllerDeploymentList := &gardencorev1.ControllerDeploymentList{}
 			g.Expect(virtualClusterClient.Client().List(ctx, controllerDeploymentList)).To(Succeed())
-			g.Expect(controllerDeploymentList.Items).To(ContainElement(MatchFields(IgnoreExtras, Fields{"ObjectMeta": MatchFields(IgnoreExtras, Fields{"Name": Equal("provider-local")})})))
+			g.Expect(
+				controllerDeploymentList.Items).To(ContainElement(MatchFields(IgnoreExtras, Fields{"ObjectMeta": MatchFields(IgnoreExtras, Fields{"Name": Equal("provider-local")})})))
+		}).Should(Succeed())
+
+		By("Verify 'gardener-system-public' namespace and 'gardener-info' configmap exist")
+		Eventually(func(g Gomega) {
+			namespace := &corev1.Namespace{}
+			g.Expect(virtualClusterClient.Client().Get(ctx, client.ObjectKey{Name: gardencorev1beta1.GardenerSystemPublicNamespace}, namespace, &client.GetOptions{})).To(Succeed())
```

Comment:
```suggestion
			g.Expect(virtualClusterClient.Client().Get(ctx, client.ObjectKey{Name: gardencorev1beta1.GardenerSystemPublicNamespace}, &corev1.Namespace{})).To(Succeed())
```

Why do you add empty `GetOptions`?